guides/source/working_with_javascript_in_rails.md

```diff
@@ -148,6 +148,7 @@ and Rails has got your back in those cases.
 Because of Unobtrusive JavaScript, the Rails "Ajax helpers" are actually in two
 parts: the JavaScript half and the Ruby half.
 
+Unless you have disabled the Asset Pipeline,
 [rails.js](https://github.com/rails/jquery-ujs/blob/master/src/rails.js)
 provides the JavaScript half, and the regular Ruby view helpers add appropriate
 tags to your DOM. The CoffeeScript in rails.js then listens for these
```

Comment:
Completely unrelated point, but is this sentence out of date?  It looks like rails.js is now pure JS rather than generated by CoffeeScript? Or did I miss something?